releasetasks/templates/firefox/release_graph.yml.tmpl

```diff
@@ -45,7 +45,6 @@ scopes:
   - docker-worker:*
   - scheduler:*
   - project:releng:signing:format:mar
-  - project:releng:signing:cert:{{ signing_class }}
```

Comment:
Actually I removed it as it was duplicated with https://github.com/mozilla/releasetasks/blob/master/releasetasks/templates/firefox/release_graph.yml.tmpl#L42